script.js

```diff
@@ -42,8 +42,16 @@ class BookCollection {
         const li = document.createElement('li');
         li.textContent = `${book.title} by ${book.author}`;
 
+        // Add alternating background colors to the list items
```

Comment:
- [optional] Great job with the comments as they help explain what you are doing, but please do as well to limit it and comment just important aspects. The comments are too many and kind of make your code not to really look clean.